lib/puppet_x/sqlserver/sql_connection.rb

```diff
@@ -1,110 +1,112 @@
-module PuppetX::Sqlserver
-  CONNECTION_CLOSED = 0
-
-  class SqlConnection # rubocop:disable Style/Documentation
-    def open_and_run_command(query, config)
-      begin
-        open(config)
-        execute(query)
-      rescue win32_exception => e
-        return ResultOutput.new(true, e.message, @connection)
-      ensure
-        close
+module PuppetX # rubocop:disable Style/ClassAndModuleChildren
```

Comment:
The changes in this file were due to rubocop, I'm guessing?

Reply:
Yep